.github/workflows/release-stable.yaml

```
@@ -0,0 +1,176 @@
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.ref }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+
+      - name: Determine tag or commit
+        uses: haya14busa/action-cond@v1
+        id: refortag
+        with:
+          cond: ${{ startsWith(github.ref, 'refs/tags/') }}
+          if_true: ${{ github.ref }}
+          if_false: latest
+      - name: Determine image tag
+        id: imagetag
+        run: echo "::set-output name=value::${TAG_OR_BRANCH##*/}"
+        env:
+          TAG_OR_BRANCH: ${{ steps.refortag.outputs.value }}
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.CR_PAT }}
+
+      - name: Build banzai-cli
+        uses: docker/build-push-action@v2
+        with:
+          tags: |
+            ghcr.io/banzaicloud/banzai-cli:latest
+            ghcr.io/banzaicloud/banzai-cli:${{ steps.imagetag.outputs.value }}
+          file: Dockerfile
+          push: true
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache
+          build-args: VERSION=${{ steps.imagetag.outputs.value }}
+
+  release-stable:
+    name: Stable release
+    runs-on: ubuntu-latest
+    needs: build
+
+    env:
+      GO_VERSION: 1.14
+      GOFLAGS: -mod=readonly
+      GO111MODULE: "on"
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup Go
+        uses: actions/setup-go@v3.2.1
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - name: Update package registry
+        run: sudo apt update --yes
+
+      - name: Install RPM
+        run: sudo apt install rpm --yes
+
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        run: make release
+
+  cdn-invalidate:
+    name: Invalidate cloudfront edge cache
+    runs-on: ubuntu-latest
+    needs: release-stable
+
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
+
+      - name: Invalidate cloudfront edge cache
+        run: |
+          aws cloudfront create-invalidation \
+            --distribution-id ${{ secrets.CDN_DISTRIBUTION_ID }} \
+            --paths "/downloads/banzai-cli/*"
+
+  trigger-docs-update:
+    name: Update docs
+    runs-on: ubuntu-latest
+    needs: release-stable
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Trigger
+        run: scripts/trigger-docs-update.sh ${{ github.ref_name }}
```

`banzaicloud.github.io` also moved to gha, so this script needs to be updated as well. I'll push a commit to this PR with the followup.
```suggestion
        run: scripts/trigger-docs-update.sh ${{ github.ref_name }}
        env:
          GITHUB_TOKEN: ${{ secrets.CR_PAT }}
```

---

Thanks for noticing!